src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -339,13 +300,12 @@ def test_fw_binaryoperator_unknown(monkeypatch):
     # Remove sign() from the list of supported binary operators
     monkeypatch.delitem(Fparser2ASTProcessor.binary_operators, "sign")
     # Generate Fortran from the PSyIR schedule
-    fvisitor = FortranWriter()
     with pytest.raises(VisitorError) as excinfo:
-        _ = fvisitor(schedule)
+        _ = fort_writer(schedule)
```

Comment:
Is this assignment to _ actually necessary? Can't we simply ignore the result? Same for other uses like this further below.

Reply:
I'm going to leave this for now and we can discuss at Weds telco. My vote is to keep them as I think it makes it clear what would be returned if the function were successful or required.